src/extension.ts

```diff
@@ -19,7 +25,7 @@ interface Snippet {
   description: string
 }
 
-let registerCompletionProvider: Disposable | undefined
+let disposable: Disposable | undefined
```

Comment:
Isn't `disposable` rather abstract? How about if we called it something like `registeredPSCompletionProvider`. When you ask me, I'm for verbosity.

Reply:
I used this since extension is simple, but I'd also prefer more descriptive name, but couldn't come up with one.

Reply:
That's just a every day struggle. We'll get the naming right down the road. Just wanted us to keep that in mind.